### DISCUSSION TITLE
Import RemoteDesktopManager CSV, multiple windows, password vault

### DISCUSSION THREAD

**Discussion by vdias** [1Remote/1Remote#625] [Ideas]
- Import from RemoteDesktopManager csv
- Open RDP External, multiple windows not TAB inside app
- User/Password vault, so you just point to the vault and no need to edit connections on user/password changes

**Comment by itagagaki** [comment]
> Import from RemoteDesktopManager csv

Do you mean Devolutions Remote Desktop Manager?

If so, it seems to be able to export a list of host, port, description, display name, and folder as a CSV, right?
(There seem to be many other export options, but the above format is probably the only one that can be used).

It would not be difficult to implement the ability to import from this CSV, but would there really be a demand for it?
I am not the owner of 1Remote, but if I get enough votes for this comment, maybe I can try.

**Comment by itagagaki** [comment]
> Open RDP External, multiple windows not TAB inside app

This would be solved when #790 is done.

**Comment by itagagaki** [comment]
> User/Password vault, so you just point to the vault and no need to edit connections on user/password changes

Devolutions Remote Desktop Manager seems to have this feature, right?

However, this requires careful consideration of security issues and may be a bit difficult. I think it would be better to consider using it in conjunction with other password management tools.
This is just my personal opinion.

**Reply by VShawn** [reply]
I don't know why I missed this topic. But the `password vault` feature was discussed a long time ago, and we ended up with the bulk editing feature over the password vault feature: https://github.com/1Remote/1Remote/issues/83

Then, the process of batch management of usernames and passwords is:

- Tag all devices with the same account with a same label
- Filter these configurations by the label.
- Select all and batch editing.
